packages/client/src/index.test.ts

```diff
@@ -181,7 +193,18 @@ describe('LogtoClient', () => {
         storage,
       });
       await logto.loginWithRedirect(REDIRECT_URI, jest.fn());
-      expect(logto.isLoginRedirect(REDIRECT_URI)).toBeFalsy();
+      expect(logto.isLoginRedirect(REDIRECT_CALLBACK_WITHOUT_CODE)).toBeFalsy();
```

Comment:
Hard to tell the difference between `REDIRECT_URI` and `REDIRECT_CALLBACK_WITHOUT_CODE`, maybe add a helper method to rescue

Reply:
TBD: @xiaoyijun 

I think we can see the constants on the top of the test file to easily learn how to tell their differences.

Is it excessive packaging to add `buildRedirectCallbackUri`?

History

- I used to use `${REDIRECT_URI}?state=${STATE}` here.
- I think it is not self-descriptive, so I replace it with a new const `REDIRECT_CALLBACK_WITHOUT_CODE`.

Reply:
Fixed.

Better: Inline `REDIRECT_CALLBACK_WITHOUT_CODE` and `REDIRECT_CALLBACK_WITHOUT_CODE`.